tests/e2e/example.test.e2e.js

```diff
@@ -44,6 +56,19 @@ test('Test Login Form', async ({page}, workerInfo) => {
   save_visual(page);
 });
 
+// test('Test Login Form OpenID navigation', async ({page}, workerInfo) => {
```

Comment:
Commented code should be removed but otherwise LGTM

Reply:
Have removed the commented code.